index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e520ead38a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="header">
+        <nav class="navbar">
+            <div class="web-name">Sohail</div>
+            <a href="#" class="bar-button">
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </nav>
+    </div>
+    <main>
+        <div class="details">
+            <h1>I'm Sohail Glad to see you!</h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] Kindly change the  h3 tag to a tag based on the project requirement `LET'S CONNECT` must be a link or a button. So fix this issue :wrench: